Source/WindowsSparsePackage/MSIX/AppxManifest.xml

```diff
@@ -37,469 +37,469 @@
         <desktop4:Extension Category="windows.fileExplorerContextMenus">
           <desktop4:FileExplorerContextMenus>
             <desktop5:ItemType Type="Directory">
```

Comment:
Add (in both AppxManifest.xml) something like:
```
            <!-- Prefix with "C" to work around an OS bug preventing the shell menu from appearing in the
                 context menu on some machines without it. There is no reason found why it works, but it really does -->
```

(it will be so easy in the future to remove it for "simplification" if there is no comment)